site/specs/voice.yml

```diff
@@ -6066,6 +6066,16 @@ components:
 
             between 1 and 25.
           example: 5.5
+        detectLanguage:
```

Comment:
the robot does this automagically to match the other PR I guess

Reply:
thanks, robots!

Reply:
🤖